justfile

```diff
@@ -218,6 +218,6 @@ git-pre-push: stop start
 prepare-sqlite:
     @if ! command -v cargo-sqlx &> /dev/null; then \
         echo "SQLX could not be found. Installing..." ;\
-        cargo install sqlx-cli --no-default-features --features sqlite,native-tls ;\
+        cargo install sqlx-cli --version 0.7.0-alpha.3 --no-default-features --features sqlite,native-tls ;\
```

Comment:
are we ok to use native-tls here, or should it also use rustls?  I am ok with rustls (one day i hope all our code can migrate to it), but we should double check if it will work ok with the certificates installed by the operating system.